package tech.run.planner.exceptions;

public class ParticipantNotFoundException extends RuntimeException{
}
